test-framework/security-jwt/src/main/java/io/quarkus/test/security/jwt/JwtTestResource.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.test.security.jwt;
+
+import java.util.Map;
+
+import io.quarkus.test.common.DevServicesContext;
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class JwtTestResource implements QuarkusTestResourceLifecycleManager, DevServicesContext.ContextAware {
```

Comment:
@mcruzdev We don't need it IMHO, a test factory like this one should only used if no devservice support is available but you have made it availabe

Reply:
OK, try to add it to `SmallryeJwtProcessorDevModeTest` and see if it helps